tests/logictest/README.md

```diff
@@ -161,7 +161,7 @@ select 1;
 1
 ```
 
-3. if some test has flaky failure, and you want ignore it, simply add skipped before statement query.(Remove it after problem solved)
+3. if some test has a flaky failure, and you want to ignore it, simply add skipped before statement query. (Remove it after the problem is solved)
 ```
 statement query skipped I
```

Comment:
if this is skipped, I think the `I` is useless.

complete script can't generate `I`.